descwl_coadd_task/coadd_in_cells.py

```diff
@@ -0,0 +1,66 @@
+from __future__ import annotations
+
+from typing import Mapping, Tuple
+
+import lsst.geom as geom
+import lsst.pipe.base as pipeBase
+import numpy as np
+from descwl_coadd import make_coadd_obs
+from lsst.cell_coadds import (
+    ObservationIdentifiers,
+    OwnedImagePlanes,
+    SingleCellCoaddBuilderConfig,
+    SingleCellCoaddBuilderTask,
+    singleCellCoaddBuilderRegistry,
+)
+from lsst.daf.butler import DeferredDatasetHandle
+from lsst.pex.config import Field, registerConfigurable
+from lsst.skymap import CellInfo
+
+
+class SCCBuilderConfig(SingleCellCoaddBuilderConfig):
+
+    seed = Field[int](
+        doc="Base seed for the random number generator",
+        dtype=int,
+        optional=False,
+    )
+
+
+@registerConfigurable("descCoaddBuilder", singleCellCoaddBuilderRegistry)
+class SCCBuilder(SingleCellCoaddBuilderTask):
+    """A concrete class to build single cell coadds"""
+
+    ConfigClass = SCCBuilderConfig
+
+    def run(
+        self,
+        inputs: Mapping[
+            ObservationIdentifiers, Tuple[DeferredDatasetHandle, geom.Box2I]
+        ],
+        cellInfo: CellInfo,
+    ):
+        exp_list = [_v[0].get(bbox=_v[1]) for _k, _v in inputs.items()]
+        # Any further selection/rejection of exposures should be done here.
```

Comment:
Will the input images be limited to those without edges in the coadd region?  Or is that the further selection you mention here?

Reply:
Yes, the input images will be without edges. This is handled upstream in the DM code, since that'd be common for all coadd implementations. This is where we'd include other QA related selection, say too bad of a seeing (or too good to handle).